lib/meandro/rules/unused_macros.ex

```diff
@@ -0,0 +1,116 @@
+defmodule Meandro.Rule.UnusedMacros do
+  @moduledoc """
+  Finds macros that are not used.
+  """
+
+  @behaviour Meandro.Rule
+
+  @impl Meandro.Rule
+  def analyze(files_and_asts, _options) do
+    for {file, module_asts} <- files_and_asts,
+        {_module_name, ast} <- module_asts,
+        result <- analyze_module(file, ast, files_and_asts) do
+      result
+    end
+    |> List.flatten()
+  end
+
+  @impl Meandro.Rule
+  def is_ignored?({module, macro}, {module, macro}) do
```

Comment:
You probably don't need `module` here. Just `macro` should be enough, but careful… can there be multiple versions of the same macro name, with different arguments? And if so… can we have a user that wants to ignore warnings on just one of them and not all?

Reply:
Fixed